.storybook/stories/picker.stories.mdx

```diff
@@ -0,0 +1,78 @@
+
+<Meta
+    title="Picker"
+    component={Picker}
+    decorators={[withA11y, withActions('onClick'), withKnobs]}
+    parameters={{
+        a11y: {
+            // ... axe options
+            element: '.story__container', // optional selector which element to inspect
+            config: {}, // axe-core configurationOptions (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#parameters-1)
+            options: {} // axe-core optionsParameter (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter)
+        }
+    }}
+/>
+
+<Preview>
+    <Story name='Default'>
+        <PickerWrapper>
+            <Picker
+                isFixedWidth={boolean('isFixedWidth', false)}
+                backgroundColor={text('backgroundColor', 'white')}
+                className={select('className', ['fy', 'form-component'], '')}
+                icon={text('icon', 'calendar-alt')}
+                altText={text('altText', 'This is for accessibility purposes')}
+                iconColor={text('iconColor', 'black')}
+                borderColor={text('borderColor', '#5b616b')}
+                borderType={select('borderType', ['bottom', 'none', 'full'], 'bottom', '')}
+                iconSize={select('iconSize', ['xs', 'lg', 'sm', '1x', '2x', '3x', '4x', '5x', '6x', '7x', '8x', '9x', '10x'], 'lg', '')}
+                dropdownDirection={select('dropdownDirection', ['right', 'left'], 'right', '')} />
+        </PickerWrapper>
+    </Story>
+</Preview>
+
+<Preview>
+    <Story name='w/ Children'>
+        <PickerWrapper>
+            <Picker
+                isFixedWidth={boolean('isFixedWidth', false)}
+                backgroundColor={text('backgroundColor', 'white')}
+                className={select('className', ['fy', 'form-component'], '')}
+                icon={'none'}
+                altText={text('altText', 'This is for accessibility purposes')}
+                iconColor={text('iconColor', 'black')}
+                borderColor={text('borderColor', '#5b616b')}
+                borderType={select('borderType', ['bottom', 'none', 'full'], 'none', '')}
+                iconSize={select('iconSize', ['xs', 'lg', 'sm', '1x', '2x', '3x', '4x', '5x', '6x', '7x', '8x', '9x', '10x'], 'lg', '')}
+                dropdownDirection={select('dropdownDirection', ['right', 'left'], 'right', '')}>
+                <FontAwesomeIcon icon="share-alt" size="lg" color="black" />
+            </Picker>
+        </PickerWrapper>
+    </Story>
+</Preview>
+
+<Props of={Picker} />
+
+
+## This is Designed to be a [Controlled Component](https://reactjs.org/docs/forms.html#controlled-components)
+Which means the `selectedOption` and `onClick` are defined inside a parent component; in this case, `PickerWrapper`.
+
+## SortFn Prop
+This prop is a function which allows for a custom sort; however, it is only invoked after the selectedOption has been sorted to the highest index. The current design of this component requires this logic!
+
```

Comment:
Can we add a comment about the default icon.

Reply:
I'm gonna just make icon an element.